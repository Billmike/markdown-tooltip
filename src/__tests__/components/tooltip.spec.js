import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Tooltip from '../../components/tooltip';

describe('Tooltip component', () => {
  test('it renders the tooltip component', () => {
    const wrapper = shallow(<Tooltip />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
