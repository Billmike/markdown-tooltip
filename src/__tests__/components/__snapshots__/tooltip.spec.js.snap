// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip component it renders the tooltip component 1`] = `
<div
  className="tooltip-container"
>
  <div
    className="tooltip-option"
    key="bold"
  >
    <button
      className="tooltip-option-btn tooltip-option-btn-bold"
      onClick={[Function]}
    >
      B
    </button>
  </div>
  <div
    className="tooltip-option"
    key="italics"
  >
    <button
      className="tooltip-option-btn tooltip-option-btn-italics"
      onClick={[Function]}
    >
      i
    </button>
  </div>
  <div
    className="tooltip-option"
    key="quotes"
  >
    <button
      className="tooltip-option-btn tooltip-option-btn-quotes"
      onClick={[Function]}
    >
      Q
    </button>
  </div>
  <div
    className="tooltip-option"
    key="largeText"
  >
    <button
      className="tooltip-option-btn tooltip-option-btn-largeText"
      onClick={[Function]}
    >
      T
    </button>
  </div>
  <div
    className="tooltip-option"
    key="smallText"
  >
    <button
      className="tooltip-option-btn tooltip-option-btn-smallText"
      onClick={[Function]}
    >
      T
    </button>
  </div>
  <div
    className="tooltip-caret"
  />
</div>
`;
